src/rest/endpoints.ts

```diff
@@ -1088,11 +1098,8 @@ The `emoji` must be [URL Encoded](https://en.wikipedia.org/wiki/Percent-encoding
   /**
    * Creates a template for the guild. Requires the `MANAGE_GUILD` permission. Returns the created [template](#DOCS_RESOURCES_TEMPLATE/template-object) object on success.
    */
-  async createGuildTemplate(
-    guildId: string,
-    payload: any
```

Comment:
isn't this technically a breaking change?

Reply:
breaking as in removing `payload` as argument

Reply:
Oop, actually I thought it took no params but it does, I'll add back